lib/rubocop/cop/chef/deprecation/chef_handler_recipe.rb

```diff
@@ -37,15 +37,12 @@ class ChefHandlerRecipe < Cop
             (send nil? :include_recipe (str {"chef_handler" "chef_handler::default"}))
           PATTERN
 
+          extend AutoCorrector
```

Comment:
I'd prefer to have the extends right up with the class so it's clear that this class extends AutoCorrector

Reply:
Agreed. Fixed.